### ISSUE TITLE
vitest 0.12.x incompatible with remix.run / @web-std/file

### ISSUE THREAD

**Issue by amuttsch** [vitest-dev/vitest#1286]
### Describe the bug

vitest `0.12` seems to add an issue when importing `import { Blob as NodeBlob, File as NodeFile } from "@web-std/file";`, which the remix.run test setup uses. (https://github.com/remix-run/remix/blob/main/packages/remix-node/globals.ts):

```
TypeError: Class extends value does not have valid prototype property undefined
 ❯ Object.<anonymous> node_modules/@web-std/file/dist/src/file.cjs:59:15
 ❯ Object.<anonymous> node_modules/@web-std/file/dist/src/lib.node.cjs:6:12
```

This causes the tests to fail. It seems like this is a regression with the 0.12 release, downgrading vitest to 0.11 makes the tests run again.

### Reproduction

Install the remix.run indie stack with `npx create-remix --template remix-run/indie-stack`, edit `package.json` and set vitest to `^0.12.4`, run `npm i` and `npm test`.

The remix `installGlobals` is imported in the vitest testup file located under `test/setup-test-env.ts`.

### System Info

```shell
System:
    OS: macOS 12.3.1
    CPU: (10) arm64 Apple M1 Max
    Memory: 1.73 GB / 64.00 GB
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 16.14.2 - ~/Library/Caches/fnm_multishells/50575_1651571166571/bin/node
    Yarn: 1.22.18 - /opt/homebrew/bin/yarn
    npm: 8.5.0 - ~/Library/Caches/fnm_multishells/50575_1651571166571/bin/npm
  Browsers:
    Firefox: 100.0
    Safari: 15.4
  npmPackages:
    @vitejs/plugin-react: ^1.3.0 => 1.3.2
    vite: ^2.9.1 => 2.9.8
    vitest: ^0.12.4 => 0.12.4
```

### Used Package Manager

npm

### Validations

- [X] Follow our [Code of Conduct](https://github.com/vitest-dev/vitest/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guidelines](https://github.com/vitest-dev/vitest/blob/main/CONTRIBUTING.md).
- [X] Read the [docs](https://vitest.dev/guide/).
- [X] Check that there isn't [already an issue](https://github.com/vitest-dev/vitest/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/vitest-dev/vitest/discussions) or join our [Discord Chat Server](https://chat.vitest.dev).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by hornta** [comment]
I have the exact same error message when importing the `user-event` package from testing-library.

`Class extends value does not have valid prototype property undefined`

**Comment by MichaelDeBoey** [comment]
@sheremet-va This problem is still present

https://github.com/remix-run/indie-stack/runs/6536194156?check_suite_focus=true
https://github.com/remix-run/blues-stack/runs/6536184942?check_suite_focus=true
https://github.com/remix-run/grunge-stack/runs/6536186808?check_suite_focus=true

**Comment by sheremet-va** [comment]
> @sheremet-va This problem is still present
> 
> https://github.com/remix-run/indie-stack/runs/6536194156?check_suite_focus=true https://github.com/remix-run/blues-stack/runs/6536184942?check_suite_focus=true https://github.com/remix-run/grunge-stack/runs/6536186808?check_suite_focus=true

We haven't released a new version yet.

**Comment by MichaelDeBoey** [comment]
Oh I'm sorry, I thought you had 🙈

**Comment by sheremet-va** [comment]
> Oh I'm sorry, I thought you had 🙈

Understandable. GitHub closes issues on PR merge, and not on releases.

> Any ETA on the release?

Today, in an hour.

**Comment by sheremet-va** [comment]
@MichaelDeBoey released in [0.12.8](https://github.com/vitest-dev/vitest/releases/tag/v0.12.8)

**Comment by MichaelDeBoey** [comment]
Seems to be fixed indeed, thanks!
